org.gnome.Evince.json

```diff
@@ -164,16 +227,24 @@
             "buildsystem": "meson",
             "config-opts": [
                 "-Dnautilus=false",
-                "-Dthumbnailer=false"
+                "-Dgtk_doc=false",
+                "-Dintrospection=false",
+                "-Dthumbnailer=false",
+                "-Duser_doc=false"
             ],
             "sources": [
                 {
-                    "type": "archive",
-                    "url": "https://download.gnome.org/sources/evince/3.38/evince-3.38.2.tar.xz",
-                    "sha256": "27d419d5fed6305e074628edcfde0cb734fffda205d63cac323391c04903bd94",
-                    "git-init" : true
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/evince.git",
+                    "tag": "3.39.2",
```

Comment:
This should be the stable version. Not the development one. Users who wants to test the development version should install it from gnome-nightly (or whatever name it is currently).

Reply:
Well, isn't it the latest stable version (Release)? At least they tagged it this way, so as their https://gitlab.gnome.org/GNOME/evince/-/blob/master/org.gnome.Evince.appdata.xml.in

Reply:
Odd minor releases are development versions. 3.39.X releases are all development releases for version 3.40 (which will actually be called version 40). This isn't a stable version.

Not sure why you would think that German, who's one of the upstream maintainers for evince would get that wrong.